README.md

```diff
@@ -62,28 +62,34 @@ The easiest way to compile the code and run the system locally is using [Docker]
 
 Don't forget to run the docker daemon!
 
-## Build the container
-
-```terminal
-$ cd opencbdc-tx                 # change to the project directory
-$ sudo -s                        # open a root shell (needed for docker)
-# docker build . -t opencbdc-tx  # build the container
-```
-
 ## Launch the System
 
 **Note:** You will need to both run the system and interact with it; you can either use two shells, or you can add the `--detach` flag when launching the system (note that it will then remain running till you stop it, e.g., with `docker stop`).
 Additionally, you can start the atomizer architecture by passing `--file docker-compose-atomizer.yml` instead.
 
+_The commands below will build a new image every time that you run it.
+You can remove the `--build` flag after the image has been built to avoid rebuilding.
+To run the system with our pre-built image proceed to the [next section](#launch-the-system-with-a-pre-built-image) for the commands to run._
+
 1. Run the System
    ```terminal
-   # docker-compose --file docker-compose-2pc.yml up
+   # docker compose --file docker-compose-2pc.yml up --build
    ```
 1. Launch a container in which to run wallet commands (use `--network atomizer-network` instead of `--network 2pc-network` if using the atomizer architecture)
    ```terminal
    # docker run --network 2pc-network -ti opencbdc-tx /bin/bash
    ```
 
+## Launch the System With a Pre-built Image
```

Comment:
I think there's one more change here I'll request before merging. With this change, a user following the document will skip a step if they choose to pull images rather than build them (namely, they'll skip running a second container where they'd run the commands in the following steps).

I think there are two solutions to that and I would be fine with either:

* Separate out the second step in *Launch the System* into its own section, and combine this section with *Launch the System* as two different ways you could launch it
* Fold this section into *Launch the System* as a note underneath the first step as an alternative

I think the first option I mention is likely to be cleaner, but I'd be happy with either.

Reply:
Good catch. I'm going to duplicate that second step in the _Launch the System With a Pre-Built Image_ section since the command is slightly different given the different image that will be run.